setup.py

```diff
@@ -33,7 +33,8 @@
         'networkx==3.1',
         'dependency-injector==4.41.0',
         # Do not upgrade pygraphviz unless security issues because it is heavily dependent on the underlying OS
-        'pygraphviz==1.10'
+        'pygraphviz==1.10',
+        'google-re2==1.0'
```

Comment:
I would let the `pygraphviz` lib at the end of the list to avoid possible confusion with the not-upgrade comment before.

Reply:
Agreed. I modify it